x/auth/stdtx.go

```diff
@@ -3,14 +3,19 @@ package auth
 import (
 	"encoding/json"
 	"fmt"
+	"math"
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/multisig"
 )
 
-var _ sdk.Tx = (*StdTx)(nil)
+var (
+	_ sdk.Tx = (*StdTx)(nil)
+
+	maxGasWanted = uint64(math.Pow(2, 63)) - 1
```

Comment:
Can you do this with bitshifts instead of math.Pow?

Reply:
Sure!

Reply:
Updated.